src/core/core/network.cpp

```diff
@@ -485,17 +482,22 @@ namespace dsn
     
     ////////////////////////////////////////////////////////////////////////////////////////////////
     network::network(rpc_engine* srv, network* inner_provider)
-        : _engine(srv), _client_hdr_format(NET_HDR_DSN)
+        : _engine(srv), _client_hdr_format(NET_HDR_DSN), _raw_msg_header_format(NET_HDR_INVALID)
     {   
         _message_buffer_block_size = 1024 * 64;
         _max_buffer_block_count_per_send = 64; // TODO: windows, how about the other platforms?
         _send_queue_threshold = (int)dsn_config_get_value_uint64(
             "network", "send_queue_threshold",
             4 * 1024, "send queue size above which throttling is applied"
             );
-        _raw_message_parser_enabled = dsn_config_get_value_bool(
-            "network", "raw_message_parser_enabled", false,
-            "whether enable raw message parser for unknown message header");
+
+        _raw_msg_header_format = network_header_format::from_string(
+            dsn_config_get_value_string(
+                "network", 
+                "raw_msg_header_format", 
```

Comment:
Is it better to change the config name to "unknown_message_header_format"? because raw_message_parser is just one kind of parser to handle unknown message header.